src/engine/TPA.cc

```diff
@@ -1691,6 +1694,40 @@ InvalidityWitness TransitionSystemNetworkManager::computeInvalidityWitness() con
     return InvalidityWitness::fromErrorPath(path, graph);
 }
 
+ValidityWitness TransitionSystemNetworkManager::computeValidityWitness() {
+    if (not isTransitionSystemDAG(graph)) { return {}; }
```

Comment:
This should always be true, no? We can `assert` it if needed.

Reply:
Changed to assert